tests/db_functions/tests.py

```diff
@@ -547,3 +547,74 @@ def test_now(self):
             ['How to Time Travel'],
             lambda a: a.title
         )
+
+    def test_length_transform(self):
+        try:
+            CharField.register_transform(Length, 'length')
```

Comment:
Lets just make sure nobody is using this outside tests. It doesn't work properly in multithreaded environment. Maybe the proper place would be under django.test?